### DISCUSSION TITLE
Passing data stored in Redux to mentions extension

### DISCUSSION THREAD

**Discussion by sebpowell** [ueberdosis/tiptap#2713] [General Questions]
Hello 👋 

I'm trying to work out how to pass in a user list to this component that's stored in Redux, but not having much luck. The example uses a hard-coded list, wondering if there's another example somewhere that shows how to do this?

Any help would be greatly appreciated. 

Thanks!

**Comment by evanfuture** [comment]
@sebpowell This is what I have on my project.  Hope it helps you figure things out: https://github.com/evanfuture/tiptaptop-extension-mention

**Comment by sebpowell** [comment]
@evanfuture – apologies for the late reply – that's great, thank you! I had a look at your sample code, and it makes sense to me, I'm going to try implementing next week, thanks for your help.
